### ISSUE TITLE
Python interface broken in v0.4

### ISSUE THREAD

**Issue by elcombato** [Breakthrough/PySceneDetect#27]
Hi,
since the methods `detect_scenes()`  and `detect_scenes_files()` expect a `SceneManager`, the [example from the docs](http://pyscenedetect.readthedocs.io/en/latest/reference/python-api/) for the Python interface is not working anymore by passing lists of scenes and detectos:
```python
video_framerate, frames_read = scenedetect.detect_scenes_file(path, scene_list, detector_list)
```

**Comment by amirmazaheri1990** [comment]
Hi,

I was wondering if you have found a workaround for this issue.

**Comment by Breakthrough** [comment]
Hi @amirmazaheri1990 / @elcombato ;

See my comment on the following issue for a temporary workaround and status update:
https://github.com/Breakthrough/PySceneDetect/issues/32#issuecomment-309627736

Thanks for your patience.

**Comment by Breakthrough** [comment]
Hello all;

Note that the following release of PySceneDetect (v0.5) will have a stable public API, implementing all the proper methods for the SceneManager class, as well as including some example scripts to help people get started more quickly.  You can see [a preview of the new API here](https://github.com/Breakthrough/PySceneDetect/blob/master/api_test.py), but note that some functionality is still subject to change before the actual release of v0.5.

Until that time, feel free to request any specific changes to the API, or to provide some feedback on the "look and feel" of the new API ([see `api_test.py`](https://github.com/Breakthrough/PySceneDetect/blob/master/api_test.py) for an example of how the new API will function).

**Comment by Breakthrough** [comment]
@KaiStritzelberger See the [main project page/README](https://github.com/Breakthrough/PySceneDetect/) on Github for an update regarding the progress on v0.5 (I am planning on releasing it by the end of the month, there are major changes even to the files I mentioned in the previous comment).  It should be out by the end of the month.

**Comment by Breakthrough** [comment]
Hello all;

v0.5-beta has been released with a much improved API (v0.5 should be out this month).  Feel free to start using the beta version to get a feel for the new refactored API; issues/bug-reports may be provided for the beta version.

Thank you all for your comments and feedback, and for helping to make PySceneDetect better! 👍
